pallets/api/src/fungibles/mod.rs

```diff
@@ -252,8 +318,7 @@ pub mod pallet {
 			admin: AccountIdOf<T>,
 			min_balance: BalanceOf<T>,
 		) -> DispatchResult {
-			let admin = T::Lookup::unlookup(admin);
-			AssetsOf::<T>::create(origin, id.into(), admin, min_balance)
+			AssetsOf::<T>::create(origin, id.into(), T::Lookup::unlookup(admin), min_balance)
```

Comment:
Do we want to know if somebody creates an asset through Pop API? If yes, can please add a simple event here. Otherwise I am fine if you leave it as is.

Reply:
Good point.

Reply:
Strictly speaking, these are just dispachables and therefore callable from anywhere. There is no guarantee that the call actually came from a contract, but may be a high probability.

From an indexing perspective, if we can correlate a contract Called event with Created event with same transaction then we're fine. One would assume so.